### PR TITLE
doc: integration test for polkadot payout

### PR DESCRIPTION
**Pull request by Prathiksha-Nataraja**
# Pull Request Template

## Description
documentation on integration test for polkadot payout and update readme

## Checklist:

- [x] Pull Request follow only a single responsibility.
- [x] Code follows the style guidelines of this project
- [x] Performed a self-review of my own code
- [x] Commented my code, particularly in hard-to-understand areas
- [x] Made corresponding changes to the documentation
- [x] Changes generate no new warnings
- [ ] Added tests that prove fix is effective or the feature works
- [ ] New and existing unit tests pass locally with the changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Checked the code and corrected any misspellings

### CODE REVIEW

File: docs/integration-testing-for-payout-notify.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+1. Make sure while the aurras is up and running
+2. Make sure [wsk cli](https://github.com/apache/openwhisk-cli) is added to the path
+3. Install the [actions](../../../#installation)
+4. Workflow generation with composer:
+   Once your YAML configuration is ready, hand it over to the composer. This composer leverages the configuration to produce a WebAssembly (wasm) file. This wasm file encapsulates the workflow based on your specified criteria. 
+   Generating a wasm file `ex: output.wasm` from the [PayoutNotification.yaml](../workflow/examples/PayoutNotification.yaml)
+   
+   ```
+   docker run -i hugobyte/workflow-composer:v0.2 generate > output.wasm
```
Comment (shanithkk) [inline docs/integration-testing-for-payout-notify.md]: wrong command, add config file

File: examples/substrate-push-notification/images/Screen-6.png
Comment (shanithkk) [inline examples/substrate-push-notification/images/Screen-6.png:1]: remove the unnecessary  screenshot, add registration screenshot

File: docs/integration-testing-for-payout-notify.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    ```
+
+12. Navigate to examples/susbtrate-push-notification
+13. Add [API configuration](../examples/substrate-push-notification/#api-configuration) and [Firebase Push Notification Configuration](../examples/substrate-push-notification/#push-notification-configuration)
+14. Install Node Dependencies using `yarn install`
+15. Start susbtrate-push-notification using `npx yarn start`
+> For Brave brower enable `Use Google services for push messaging` using brave://settings/privacy
+16. Select the account for which polkadot notification to be received
+17. Click Register Balance Notification button
+18. Select the event source, here in this case `invoker`. Copy the Push notification token(user device token).
```
Comment (shanithkk) [inline docs/integration-testing-for-payout-notify.md]: change invoker name

File: docs/integration-testing-for-payout-notify.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+   zip -r - Cargo.toml src output.wasm | docker run -e RELEASE=true -i --rm hugobyte/openwhisk-runtime-rust:v0.3 -compile main > output.zip
+   ```
+
+6. Creating the action `payout_notify`
+   
+   ```
+   wsk -i action create payout_notify  output.zip --docker hugobyte/openwhisk-runtime-rust:v0.3 --timeout 300000 --web true --param allowed_hosts "<allowed_hosts>"
+   ```
+
+7. Register event source using the below command with name as param eg: --name invoker.
```
Comment (shanithkk) [inline docs/integration-testing-for-payout-notify.md]: chainge the name

File: docs/integration-testing-for-payout-notify.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+   
+   ```
+   ./register_event_source_polkadot.sh --name invoker --openwhiskApiHost <API_HOST>
```
Comment (shanithkk) [inline docs/integration-testing-for-payout-notify.md]: add proper name

**Review by shanithkk — CHANGES_REQUESTED** [review]

**Review by Mr-Nobody21 — APPROVED** [review]

**Review by shanithkk — APPROVED** [review]

**Pull request merged**